src/vs/workbench/services/editor/browser/editorOverrideService.ts

```diff
@@ -375,13 +370,13 @@ export class EditorOverrideService extends Disposable implements IEditorOverride
 		return out;
 	}
 
-	private async doHandleConflictingDefaults(editorName: string, currentEditor: IContributedEditorInput, options: IEditorOptions | undefined, group: IEditorGroup) {
+	private async doHandleConflictingDefaults(resource: URI, editorName: string, currentEditor: IContributedEditorInput, options: IEditorOptions | undefined, group: IEditorGroup) {
```

Comment:
I hope this does not break any semantics of how it used to be where we got the `resource` from `currentEditor`